### DISCUSSION TITLE
Chrome extensions

### DISCUSSION THREAD

**Discussion by ApoNikol** [seleniumbase/SeleniumBase#3022] [General]
Hello, and thanks for creating this amazing library!
I am wondering if i could add somehow a chrome extension to the default chrome user of uc mode, and if yes how.
Also would i get detected because i changed the default settings of this chrome user?

**Comment by mdmintz** [comment]
Use the `extension_dir` arg to add a Chrome extension directory (need to use the directory format instead of the zip format when using UC Mode). https://github.com/seleniumbase/SeleniumBase/blob/538b5d162c313f842f85af80992de49dd4f44889/seleniumbase/plugins/sb_manager.py#L74

**Comment by ApoNikol** [comment]
Thank you it worked, saved me a lot of time!
